main.go

```diff
@@ -155,6 +155,13 @@ func loadConfigs() (config.Group, error) {
 		}
 	}
 
+	// Command line Arg
+	if flag.NArg() > 0 {
+		confString := strings.Join(flag.Args(), "\n")
+		confBody := ":" + config.Port + "\n" + strings.Replace(confString, ",", "\n", -1)
```

Comment:
... Then next line prepends the default port listen and you're good to go. No need for commas. :)